cli/Cargo.toml

```diff
@@ -75,3 +75,5 @@ version = "1.0"
 version = "1.21"
 features = ["rt"]
 
+[dependencies.num-bigint]
```

Comment:
@howardwu please check if you don't mind an extra dependency

Reply:
Thanks for checking. We technically use `num-bigint` to impl `Display` in `snarkvm::utilities::BigInteger256/384`,  so the extra dependency isn't a big deal.